drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/update_unit_availability_job.rb

```diff
@@ -40,7 +40,7 @@ def perform(force: false)
         end
       end
 
-      handle_alert("Failed to sync #{failed_updates} capacity updates to LINK.")
+      handle_alert("Failed to sync #{failed_updates} capacity updates to LINK.") unless failed_updates.zero?
```

Comment:
Is `failed_updates` always an integer?  If so, maybe:
```suggestion
      handle_alert("Failed to sync #{failed_updates} capacity updates to LINK.") if failed_updates.positive?
```

If it can be `nil` this probably wants an `&`

Reply:
Yes it's initialized to zero